src/main/java/edu/greenblitz/robotName/subsystems/gyros/SimulationGyro.java

```diff
@@ -1,34 +1,36 @@
 package edu.greenblitz.robotName.subsystems.gyros;
 
+
 import edu.greenblitz.robotName.RobotConstants;
 import edu.greenblitz.robotName.subsystems.swerve.chassis.SwerveChassis;
 import edu.wpi.first.math.geometry.Rotation2d;
 
 public class SimulationGyro implements IAngleMeasurementGyro {
-	
-	private GyroInputsAutoLogged lastInputs = new GyroInputsAutoLogged();
-	
-	@Override
-	public void updateYaw(Rotation2d yaw) {
-		lastInputs.yaw = yaw.getRadians();
-	}
-	
-	@Override
-	public void updatePitch(Rotation2d pitch) {
-		lastInputs.pitch = pitch.getRadians();
-	}
-	
-	@Override
-	public void updateRoll(Rotation2d roll) {
-		lastInputs.roll = roll.getRadians();
-	}
-	
-	@Override
-	public void updateInputs(GyroInputsAutoLogged inputs) {
-		inputs.roll = 0;
-		inputs.pitch = 0;
-		inputs.yaw += (SwerveChassis.getInstance().getChassisSpeeds().omegaRadiansPerSecond * RobotConstants.SimulationConstants.TIME_STEP);
-		
-		lastInputs = inputs;
-	}
-}
\ No newline at end of file
+
+    private GyroInputsAutoLogged lastInputs = new GyroInputsAutoLogged();
+
+    @Override
+    public void updateYaw(Rotation2d yaw) {
+        lastInputs.yaw = yaw.getRadians();
+    }
+
+    @Override
+    public void updatePitch(Rotation2d pitch) {
+        lastInputs.pitch = pitch.getRadians();
+    }
+
+    @Override
+    public void updateRoll(Rotation2d roll) {
+        lastInputs.roll = roll.getRadians();
+    }
+
+    @Override
+    public void updateInputs(GyroInputsAutoLogged inputs) {
+        inputs.roll = 0;
+        inputs.pitch = 0;
```

Comment:
why 0?

Reply:
I dont know its from master

Reply:
Its because the gyro is only rotating on the yaw axis and the pitch and roll are unimportant and unneccesary.